routes/products.js

```
@@ -1,6 +1,115 @@
 const express = require('express');
 const routerProducts = express.Router();
-// needs to be linked to something...
-//const productDB = require('./../db/products.js');
+const knex = require('../database');
+
+routerProducts.route('/').get((req, res) => {
+  knex
+    .raw('SELECT * FROM products')
+    .then((result) => {
+      if (result.rows.length === 0) {
```

I don't think this should throw an error. You could just not have any products and that's fine.